packages/types/src/type/ExtrinsicEra.spec.ts

```diff
@@ -0,0 +1,31 @@
+// Copyright 2017-2019 @polkadot/types authors & contributors
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
+
+import ExtrinsicEra, { MortalEra } from './ExtrinsicEra';
+import U64 from '../primitive/U64';
+
+describe('ExtrinsicEra', () => {
+
+  it('decodes an Extrinsic Era with immortal', () => {
+    const extrinsicEra = new ExtrinsicEra(new Uint8Array([0,0,0]));
```

Comment:
for immortal era, the length is 1, so why pass a length 3 u8a?

Reply:
done